enterprise/admin/views.py

```diff
@@ -291,6 +291,13 @@ def _handle_bulk_upload(cls, enterprise_customer, manage_learners_form, request,
 
         return all_processable_emails
 
+    # TODO: this method is HUGE (> 100 lines) and complicated. Refactor it.
```

Comment:
@mattdrayer No, it won't be part of this PR. I've logged an internal Opencraft ticket for that, so please discuss with @bradenmacdonald if we need edX ticket or Opencraft ticket should suffice.